sdk/src/signer/mod.rs

```diff
@@ -125,12 +102,6 @@ impl PartialEq for dyn Signer {
 
 impl Eq for dyn Signer {}
 
-impl std::fmt::Debug for dyn Signer {
-    fn fmt(&self, fmt: &mut std::fmt::Formatter) -> std::fmt::Result {
-        write!(fmt, "Signer: {:?}", self.pubkey())
-    }
-}
```

Comment:
This didn't allow me to `println!(` Signer trait, I ended up [doing this](https://github.com/solana-labs/solana/pull/33860/files#r1371737483).